package.json

```diff
@@ -98,7 +98,7 @@
     "@types/chai-dom": "^1.11.0",
     "@types/enzyme": "^3.10.12",
     "@types/mocha": "^10.0.1",
-    "@types/node": "^20.1.4",
+    "@types/node": "^20.2.3",
```

Comment:
Yeah, we can set it to v14 because this is the version of `node` we use in our CI: https://github.com/mui/mui-x/blob/d6761b833b3f2ccac6c077f1c28cb7c448eec18e/.circleci/config.yml#L39

Reply:
Although we will end up with multiple versions on `@types/node` installed anyway, because other dependencies depend on it - see current lock file:
https://github.com/mui/mui-x/blob/d6761b833b3f2ccac6c077f1c28cb7c448eec18e/yarn.lock#L3002-L3010
I'm not sure which one will be picked up by VSCode.

Reply:
Or we could wait and see how https://github.com/mui/material-ui/pull/37173 unfolds and possibly lock it on v18 for the time being. 🤔

Reply:
> Although we will end up with multiple versions on `@types/node` installed anyway, because other dependencies depend on it

Ahh, yes, that specific `v14` coming from the `@fast-csv/format` a sub-dep of `exceljs` library 🙈
Honestly, I have no idea why they have `@types/node` as their dependency... I might be missing something, but it seems like an oversight/bug...